src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -33,9 +35,11 @@ internal TagHelperOutput(string tagName)
         {
             TagName = tagName;
             Attributes = new Dictionary<string, object>(attributes, StringComparer.OrdinalIgnoreCase);
+            _preElement = new DefaultTagHelperContent();
```

Comment:
Nope, it's just the pattern that we've used in Mvc/Razor since I've been on the team :smile: